README.md

```diff
@@ -99,10 +99,10 @@ cs.addCodec(new MyCodec("application/myType"));
 
 ### To use with Node.js
 
-> **Warning**: We no longer actively support Node.js version 14 and lower.
+> **Warning**: We no longer actively support Node.js version 16 and lower.
 
--   [Node.js](https://nodejs.org/) version 14+
--   [npm](https://www.npmjs.com/) version 7+
+-   [Node.js](https://nodejs.org/) version 16+
```

Comment:
Just noticed: This should probably either be 18+ or > 16 if version 16 is not supported anymore, right? 

```suggestion
-   [Node.js](https://nodejs.org/) version 18+
```